Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogScreen.h

```diff
@@ -18,8 +18,10 @@
 #include <GemCatalog/GemInspector.h>
 #include <GemCatalog/GemModel.h>
 #include <GemCatalog/GemSortFilterProxyModel.h>
+
 #include <QSet>
 #include <QString>
+#include <QStackedWidget>
```

Comment:
consider forward declaring this instead

Reply:
Forward declared most includes in the file